src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -466,3 +466,50 @@ def upload_string(to_upload, media_type=None, keep_open=False, wait_on_close=Fal
         handler.close(block=wait_on_close, **remaining_kwargs)
 
     return handler
+
+def download_folder(project, destdir, folder="/", chunksize=dxfile.DEFAULT_BUFFER_SIZE):
```

Comment:
Can this method be unified with existing code in dxpy for downloading folders? For example, "dx download -r" must do something similar, so possibly it could be expressed in terms of this method...